03-Snooping_Operations/README.md

```diff
@@ -44,8 +44,7 @@ balancer reachable from our computer. Once we know the URL, we can reach it
 with curl or with your browser.
 
 ```
-URL=http://xxx.xxx.xxx.xxx:yyyyy
-curl $URL/hello.txt
+curl "$(minikube service nginx-deployment --url)/hello.txt"
```

Comment:
I think I'd rather store it in a variable, so that any further commands are also shorter and don't need to call minikube... How about:

URL=$(minikube service nginx-deployment --url)
curl "$URL/hello.txt"